cmd/cli/main.go

```diff
@@ -56,6 +56,11 @@ var (
 	triggerDumpFile = flag.String("trigger-dump-file", "", "File that holds trigger dump JSON from api method response")
 )
 
+var (
```

Comment:
After that PR cli will contain two commands for updating: "update from-version" and "move-to-cluster-forwards". How the user should choose between them?
At the same time, the new database format will be used not only with cluster-redis but with any type of redis. So, the format of database keys is connected with version, but not with redis type. And I think that better include new migration command into "update from-version".

Reply:
I agree with your remark. But at the moment, this is a script for migrating Kontur's installation. The new release of Moira and migration script will be done before merging to master.